README.md

```diff
@@ -4,22 +4,32 @@
 
```

Comment:
Not in the current proposed changes, but should we keep the old illustrations?

Reply:
I don't feel strongly either way. Happy to remove if you think that's better.